kernel/arch/dreamcast/include/dc/sound/sound.h

```diff
@@ -216,7 +217,28 @@ void snd_adpcm_split(uint32_t *data, uint32_t *left, uint32_t *right, size_t siz
 
 /** @} */
 
+/** \brief  Get AICA channel position.
+
+    This function returns actual the channel position
+    that stores in SPU memory and updated by the SPU firmware.
+
+    \param  chn             The channel to retrieve position.
+
+    \return                 Last channel position in samples.
+*/
+uint16_t snd_get_pos(uint32_t ch);
```

Comment:
The channel doesn't really need to be `uint32_t`, I'd prefer those types to be used when we need a specific size. You could just use "unsigned int" here.